daemon/libnetwork/network_windows.go

```diff
@@ -272,13 +270,6 @@ func findResolver(resolvers []*Resolver, gw4, gw6 string) *Resolver {
 	return nil
 }
 
-func defaultIpamForNetworkType(networkType string) string {
-	if windows.IsBuiltinLocalDriver(networkType) {
```

Comment:
I think we still need this distinction - just the condition is wrong (and we may just switch to the `null` driver instead, the `windows` IPAM may stay lost).

The condition is not just whether it's a builtin-driver, but also if the host is supposed to be managed by Microsoft IPAM or if Dockers builtin IPAM should be used by default on this host.

Reply:
... forget about re including this distinction. Not necessary.